FBReader/src/main/java/org/geometerplus/android/fbreader/benetech/OptionsMenuHandler.java

```diff
@@ -163,4 +163,11 @@ public boolean onCreateOptionsMenu(Menu menu, List<PluginApi.ActionInfo> myPlugi
     public boolean onCreateOptionsMenuWithoutPluginActions(Menu menu){
         return onCreateOptionsMenu(menu,new LinkedList<PluginApi.ActionInfo>());
     }
+
+    public void hideMainScreenMenuOptions(Menu menu) {
```

Comment:
The below method can be called 4 times.
hideMainScreenMenuOption(menu,  int id) {
menu.findItem(id).setvisible(false);
}